### PR TITLE
feat: remove ffmpeg

### PR DESCRIPTION
**Pull request by chmelevskij**
Removing ffmpeg to see if it breaks anythin

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -239,94 +239,6 @@ function apps(done) {
...
-        var libDest = path.join(folder, pkg.name, arch);
-        console.log('Copy ffmpeg library to Windows app (' + libSrc + ' to ' + libDest + ')');
-        return gulp.src(libSrc)
-                   .pipe(gulp.dest(libDest));
-    }
-
-    if ((arch == 'linux32') || (arch == 'linux64')) {
-
-        // Copy Ubuntu launcher scripts to destination dir
-        var launcherDir = path.join(folder, pkg.name, arch);
```
Comment (McGiverGim) [inline gulpfile.js:289]: Maybe you have removed too much? :P

**Comment by McGiverGim** [comment]
There is no plan to remove the feature, but it seems it works without the library for @chmelevskij

**Comment by chmelevskij** [comment]
> it breaks loading and exporting video 😆

@nerdCopter did you try it and it broke? Can you provide screeshot or video?

I barely ever use it and have only small sample and it did work for me. But I'm on macOS so maybe I have h.264 encoder because of 🤔

**Comment by nerdCopter** [comment]
master:
image

`28d72b77`
open same video: 
image
export video: 
image

**Comment by haslinghuis** [comment]
@nerdCopter maybe we need to install ffmpeg and maybe additional codecs on linux?

**Comment by nerdCopter** [comment]
all 3 OS has always had ffmpeg packages since cleanflight. `./library/*/*`.
if there is a way for system-fallback to OS-level ffmpeg, that might be nice and would also have potential for non-included (non-free) codecs.  but really i have no clue if/how such is possible.  i'm surprised this PR used OSX-level codec as reported by @chmelevskij

**Comment by nerdCopter** [comment]
- related, 2017: https://tangiblejs.com/posts/enabling-proprietary-audiovideo-codecs-nw-js
- i think this the reason they are included.  but even now, newer [ffmpeg binaries](https://github.com/nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt/releases) seem to excluded some non-free codecs.  (at one time BBE could use fatshark DVR .avi files, bot no longer)

**Comment by chmelevskij** [comment]
Thanks for looking into it @nerdCopter 

From what I understand issue lies in nwjs/chromium/ffmpeg open source licences...

But I'm guessing if we move to PWA seems like chrome supports it. Though not chromium...

Given the maintenance cost for the team I think it's a valid tradeoff.

**Pull request closed without merging**